react-frontend/src/login/PreLoginDashboardComponent.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import PeerlyTextAndLogo from "shared-components/peerly-logo/PeerlyTextAndLogo";
+import InformativeTextComponent from "login/InformativeTextComponent";
+import LoginImageComponent from "login/LoginImageComponent";
+
+const Wrapper = styled.div`
+  height: 100vh;
+  background-color: ${({ theme }) =>
+    theme === "dark" ? "var(--atomic)" : "var(--white)"};
+`;
+
+const PreLoginDashboardComponent = ({ theme, className }) => (
+  <Wrapper
+    theme={theme}
+    className={`d-sm-block d-xs-block d-md-none flex-column d-flex justify-content-around ${className}`}
+    data-testid="officeInfo"
+  >
+    <div>
```

Comment:
Unnecessary div

Reply:
@mayuriardad  div is require because of peerly logo component not bind into div, so when we content-around then spilt logo and peerly text.